### PR TITLE
administration/troubleshooting: add password reset instructions

### PR DESCRIPTION
**Pull request by mchalski**
Issues: MEN-922

Signed-off-by: mchalczynski <marcin.chalczynski@rndity.com>

accompanying doc for https://github.com/mendersoftware/integration/pull/147

@maciejmrowiec @mendersoftware/rndity

### CODE REVIEW

**Comment by mchalski** [comment]
ping?

File: 06.Administration/06.Troubleshooting/docs.md [hunk 1]
```diff
@@ -200,3 +200,18 @@ user@local$ docker inspect menderproduction_mender-deployments_1 |& less
...
+## Resetting administrative credentials
+
+The web UI does not currently provide a 'remind/reset password' feature. However, if the credentials for
+the administrative account are lost, the account can be reset via a dedicated script
+in the [integration repository](https://github.com/mendersoftware/integration/blob/master/reset-user).
+
+To reset the account, simply run the script from the machine which hosts the `mender-mongo-useradm` Docker service:
+
+```
+./reset-user
```
Comment (bboozzoo) [inline 06.Administration/06.Troubleshooting/docs.md]: Is there any output printed to the console?

Comment (mchalski) [inline 06.Administration/06.Troubleshooting/docs.md]: no

Comment (mchalski) [inline 06.Administration/06.Troubleshooting/docs.md]: well, not true - mongo does emit something. I can include it
```
MongoDB shell version v3.4.1
connecting to: mongodb://127.0.0.1:27017/useradm
MongoDB server version: 3.4.1
WriteResult({ "nRemoved" : 0 })

```

Comment (mchalski) [inline 06.Administration/06.Troubleshooting/docs.md]: done

**Pull request merged**